contracts/upgradeable_contracts/arbitrary_message/BasicForeignAMB.sol

```diff
@@ -29,9 +29,7 @@ contract BasicForeignAMB is BasicAMB, MessageRelay, MessageDelivery, MessageProc
             _data,
             true
         );
-        require(
-            messageId & 0xffffffff00000000000000000000000000000000000000000000000000000000 == MESSAGE_PACKING_VERSION
-        );
+        _validateMessageId(messageId);
```

Comment:
```suggestion
        require(_isMessageVersionValid(messageId));
```

Reply:
it is not good to hide `require` within the method.

Reply:
Got it